src/js/rx/components/ErrorView.jsx

```diff
@@ -58,9 +58,10 @@ class ErrorView extends React.Component {
             status="warning"/>
         );
     }
+
     return (
       <div className="call-out rx-app-error rx-tab-explainer">
-        // needed due to call-out class css rule targeitng
+        // needed due to call-out class css rule targeting
```

Comment:
I'm not sure what this is about. As far as I can tell, `call-out` is a class only used in Facility Locator. I can remove that from the class name and this hack (if it is one) if you don't think it's needed.